include/mbedtls/platform_util.h

```diff
@@ -25,6 +25,33 @@
 #ifndef MBEDTLS_PLATFORM_UTIL_H
 #define MBEDTLS_PLATFORM_UTIL_H
 
+#ifdef mbedtls_platform_zeroize
+#error "'mbedtls_platform_zeroize' as macro should be defined in platform_util.h file"
+#endif
+
+#ifndef MBEDTLS_PLATFORM_ZEROIZE_ALT
+// Standard macros
+#ifdef _WIN32
+#include <windows.h>
+#define mbedtls_platform_zeroize RtlSecureZeroMemory
+#endif
+// Other possible implementations
+/* C11/C++11
+ *#include <strings.h>
+ *#define mbedtls_platform_zeroize( buf, len ) memset_s( (buf), (len), 0, (len) )
+ */
+/* FreeBSD
+ * #include <strings.h>
+ * #define mbedtls_platform_zeroize explicit_bzero
+ */
+#else
+/*
+ * User-defined alternative macro definition for mbedtls_platform_zeroize
+ */
+#undef mbedtls_platform_zeroize //No macro, ensure error message in platform_util.c
```

Comment:
Is this undef necessary? Are there scenarios, in which the macro may be defined but not needed? If not, then I would remove this line, as it gives impression of lack of robustness in the preprocessor definition management.

Reply:
This `#else` block (line 47) is a placeholder, and an alternative implementation of a zeroizing macro should replace the `#undef` directive.
Usage of an alternative macro is controlled elsewhere - in `config.h`, and `#undef` here should trigger an error if no implementation of the macro was provided.
Could you think of a better way to write this code?

Reply:
Maybe the "No macro, ..." comment could be elaborated a little bit into something like "This will trigger an error if no implementation has been provided". However, I don't consider changing this necessary.

Reply:
Thanks, I replaced the comment as you suggested.